src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestComma2.java

```diff
@@ -0,0 +1,28 @@
+/*
+SeparatorWrap
+tokens = COMMA
+option = nl
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+public class InputSeparatorWrapForTestComma2 {
+
+    String s;
+
+    // violation below '',' should be on a new line.'
+    int a,  
```

Comment:
You can suppress checkstyle violations for this class in https://github.com/checkstyle/checkstyle/blob/97a74f5dd52a33c0e802ec9c592c7726bf8a3aa6/config/checkstyle_resources_suppressions.xml#L1, make sure to leave a comment about why we do this.

Reply:
`mvn clean verify` is failing from trailing whitespace in test input, you must suppress these violations. Please make sure that `mvn clean verify` passes before next push.

Reply:
Done